rotkehlchen/types.py

```diff
@@ -97,6 +97,7 @@
 SPAM_PROTOCOL = 'spam'
 GEARBOX_PROTOCOL = 'gearbox'
 HOP_PROTOCOL_LP = 'hop_lp'
+UMAMI_PROTOCOL: Literal['umami'] = 'umami'
```

Comment:
you are mixing counterparty and protocol here. Undo this and use `CPT_UMAMI` everywhere. No need to introduce a longer name. If it's also used as a protocol that's fine.

And all constants should be marked with `Final`